packages/wallet/src/services/HandlesTracker.ts

```diff
@@ -108,7 +108,19 @@ export const createHandlesTracker = (
       const filteredUtxo = utxo.flatMap(([_, txOut]) =>
         uniqBy(
           [...(txOut.value.assets?.keys() || [])]
-            .filter((assetId) => handlePolicyIds.some((policyId) => assetId.startsWith(policyId)))
+            .filter((assetId) => {
+              const matchPolicyId = handlePolicyIds.some((policyId) => assetId.startsWith(policyId));
```

Comment:
This is an early exit condition.

`if(!handlePolicyIds.some((policyId) => assetId.startsWith(policyId))) return false;`

may help reducing indent level of code, maybe making it more easy to read.

Reply:
https://github.com/input-output-hk/cardano-js-sdk/pull/905/commits/a1061aea9783682cf5b6f92e6ff0135e0eea7b40